workitem/workitem_repository_blackbox_test.go

```diff
@@ -510,3 +513,159 @@ func (s *workItemRepoBlackBoxTest) TestConcurrentWorkItemCreations() {
 		assert.Equal(s.T(), 0, report.failures)
 	}
 }
+
+func (s *workItemRepoBlackBoxTest) TestList() {
+	s.T().Run("list with explicit order", func(t *testing.T) {
+		fxt := tf.NewTestFixture(t, s.DB,
+			tf.Iterations(2),
```

Comment:
We do not need `iterations(2)`

Reply:
Fixed in 6ad821e59cd8dffbb19ec35fe49dbf66a6e0ef19